.travis.yml

```diff
@@ -1,10 +1,11 @@
-language: node_js
-node_js: '8'
+language: minimal
+dist: xenial
 sudo: required
 env:
   - COVERALLS_PARALLEL=true
 services:
   - docker
+install: ./ci/install.sh
```

Comment:
If we'll have an actual install script in the future then please ignore this but if not we can just make this `install: true` and not add an empty script.

Reply:
I don't think we will need to, checking if the build will work on the `travis` branch since I had trouble with `install: false`.

Reply:
worked, will rebase